crates/op-rbuilder/src/metrics.rs

```diff
@@ -155,6 +157,35 @@ impl OpRBuilderMetrics {
     }
 }
 
+/// Gauges to check whether particular flags are enabled or not
+#[derive(Metrics, Clone)]
+#[metrics(scope = "op_rbuilder_flags")]
+pub struct FlagMetrics {
```

Comment:
We could do it even without structure, because we don't really using it
Just create a function that set all metrics with gauge! macros and metrics name